sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+- Added specification for [`sycl_ext_intel_fpga_task_sequence`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/proposed/sycl_ext_intel_fpga_task_sequence.asciidoc) extension. intel/llvm#6348
+- Added specification for [`sycl_ext_codeplay_enqueue_native_command`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_codeplay_enqueue_native_command.asciidoc) extension. intel/llvm#14136
+- Added specification for [`SPV_INTEL_bindless_images`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/design/spirv-extensions/SPV_INTEL_bindless_images.asciidoc) extension. intel/llvm#12927
+
+## Improvements
+
+### SYCL Compiler
+
+- Improved compilation flow around integration footer when no 3rd-party host
+  compiler is used. New compilation flow creates less temporary files and
+  therefore should result in a slightly faster compilation. intel/llvm#13607 intel/llvm#14402
+- Added support for `truncf`, `sinpif`, `rsqrtf`, `exp10f`, `ceilf`,
+  `copysignf`, `cospif`, `fmaxf` and `fminf` functions in SYCL kernels as part of
+  [C-CXX-StandardLibrary](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/supported/C-CXX-StandardLibrary.rst)
+  extension. intel/llvm#14132 intel/llvm#13714
+- Added support for more IMF functions as part of [C-CXX-StandardLibrary](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/supported/C-CXX-StandardLibrary.rst) extension. intel/llvm#13786
+- Added `-fsystem-debug` command line option to complement existing
+  `-fno-system-debug`. intel/llvm#13256
+- Improved wording of an error about implicit `this` capture in a kernel. intel/llvm#14100
+- Improved `--save-temps` to work with `-fsycl-host-compiler`. intel/llvm#114751
+- Improved error message about missing AMDGPU architecture when several values
+  are passed into `-fsycl-targets`. intel/llvm#13078
+- Reduced list of commands invoked to generate dependencies using `-MD` flag
+  by one command. intel/llvm#13217
+- Enhanced diagnostic emitted if CUDA target triple passed to `-fsycl-targets`
+  is incorrect. intel/llvm#14673
+- Reduced size of shadow memory used by address sanitizer to avoid running out
+  of memory in multi-GPU environments. intel/llvm#13857
+- Enhanced address sanitizer to be able to detect out-of-bounds access to
+  local accessors. intel/llvm#13503
+- Enhanced address sanitizer to detect incorrect uses of USM deallocation
+  functions (like calling `sycl::free` on a pointer that was not allocated as
+  a USM pointer). intel/llvm#12882
+- Enhanced `-fintelfpga` flag: when it is used together with `-fp-module=fast`
```

Comment:
```suggestion
- Enhanced `-fintelfpga` flag. When used together with `-fp-module=fast`
```

Reply:
Applied in 4dfea21f74ea84cd81abd20b140f02bb85bda622